<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CardEditor</class>
 <widget class="QDialog" name="CardEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CardEditor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:18px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:xx-large; font-weight:600;&quot;&gt;IC Card:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>IC Name:</string>
     </property>
     <property name="buddy">
      <cstring>name</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QLineEdit" name="name"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Description:</string>
     </property>
     <property name="buddy">
      <cstring>description</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="4">
    <widget class="QLineEdit" name="description">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Interaction Pattern:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="6">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="quiet">
       <property name="toolTip">
        <string>Quiet State</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="iceditor.qrc">
         <normaloff>:/images/quietstate.png</normaloff>:/images/quietstate.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="myselfnone">
       <property name="toolTip">
        <string>By Myself no Interaction</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="iceditor.qrc">
         <normaloff>:/images/myselfnone.png</normaloff>:/images/myselfnone.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="myselfwith">
       <property name="toolTip">
        <string>By Myself with Interaction</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="iceditor.qrc">
         <normaloff>:/images/myselfwith.png</normaloff>:/images/myselfwith.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="othersnone">
       <property name="toolTip">
        <string>By Others no Interaction</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="iceditor.qrc">
         <normaloff>:/images/othersnone.png</normaloff>:/images/othersnone.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="otherswith">
       <property name="toolTip">
        <string>By Others with Interaction</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="iceditor.qrc">
         <normaloff>:/images/otherswith.png</normaloff>:/images/otherswith.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="mixed">
       <property name="toolTip">
        <string>Mixed</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="iceditor.qrc">
         <normaloff>:/images/mixedstate.png</normaloff>:/images/mixedstate.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="6">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>My Task: </string>
       </property>
       <property name="buddy">
        <cstring>myTask</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="4">
      <widget class="QLineEdit" name="myTask"/>
     </item>
     <item row="1" column="0" colspan="4">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Time Critical Condition: </string>
       </property>
       <property name="buddy">
        <cstring>TimeCritical</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLineEdit" name="TimeCritical"/>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Name of Other IC:</string>
       </property>
       <property name="buddy">
        <cstring>otherName</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="3" colspan="2">
      <widget class="QLineEdit" name="otherName"/>
     </item>
     <item row="3" column="0" colspan="3">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Message to Other IC:</string>
       </property>
       <property name="buddy">
        <cstring>otherMsg</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="3" colspan="2">
      <widget class="QLineEdit" name="otherMsg"/>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Other IC's Task: </string>
       </property>
       <property name="buddy">
        <cstring>otherTask</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="2" colspan="3">
      <widget class="QLineEdit" name="otherTask"/>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Card</string>
     </property>
     <property name="buddy">
      <cstring>num</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QSpinBox" name="num"/>
   </item>
   <item row="5" column="2">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>of</string>
     </property>
     <property name="buddy">
      <cstring>totalNum</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QSpinBox" name="totalNum"/>
   </item>
   <item row="5" column="4" colspan="2">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>(If necessary please use several IC cards to describe an IC) </string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="iceditor.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>name</sender>
   <signal>returnPressed()</signal>
   <receiver>CardEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>description</sender>
   <signal>returnPressed()</signal>
   <receiver>CardEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>myTask</sender>
   <signal>returnPressed()</signal>
   <receiver>CardEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>201</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TimeCritical</sender>
   <signal>returnPressed()</signal>
   <receiver>CardEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>389</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>366</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>otherName</sender>
   <signal>returnPressed()</signal>
   <receiver>CardEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>309</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>153</x>
     <y>365</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>otherMsg</sender>
   <signal>returnPressed()</signal>
   <receiver>CardEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>522</x>
     <y>288</y>
    </hint>
    <hint type="destinationlabel">
     <x>92</x>
     <y>379</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>otherTask</sender>
   <signal>returnPressed()</signal>
   <receiver>CardEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>538</x>
     <y>317</y>
    </hint>
    <hint type="destinationlabel">
     <x>55</x>
     <y>362</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>accept()</slot>
 </slots>
</ui>
